ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md

```diff
@@ -0,0 +1,175 @@
+
+## What is Ansible AVD (Ansible collection for Arista Validated Designs)
+While Ansible is the automation engine, Ansible AVD is a “collection” added to Ansible. It is an open source project hosted on GitHub and maintained by Arista, letting users contribute directly. 
+A “collection” is a distribution format for Ansible content that can include roles, modules, and plugins. These topics are most likely new to you, but do not worry, they will be described in later sections. For now, just remember that a “collection” is a way to easily share and distribute Ansible data. You can install and use collections through Ansible Galaxy, which is Ansible’s official hub for sharing Ansible content.
+
+![Figure 3: Arista Leaf Spine topology](../_media/getting-started/Ansible-AVD.png)
+
+In the illustration above, the Ansible AVD collection is shown as the box with the red background on the right hand side. 
+When designing an EVPN Layer 3 Leaf Spine (L3LS) fabric you don't need to be an expert on Arista Design and Deployment Guides. All those rules and guidelines are integrated in the Ansible collection for Arista Validated Designs (AVD), or in short: Ansible AVD.<br>
+Ansible AVD is an opinionated collection, meaning that field experiences and best practices are also included. With Ansible AVD, users get up and running quickly without having to invent every part of the network configuration from scratch. This means only the most basic information must be provided by the user to get a new fabric up and running.
+ 
+Examples could be:
+- Username/password for network devices and Cloudvision
+- OOB management IP range
+- IP scope for link addresses
+- IP scope for loopback interfaces
+- VLAN and VNI ranges
+- Device hostnames
+- And so on
+
+All are details that must be decided upon in any fabric design. In a standard spine-leaf topology, Ansible AVD takes care of all the work past the theoretical design.
+
+## When to use Ansible AVD and when not to
+It’s important to mention when to, and perhaps more importantly, when not to use Ansible AVD:<br> Automating provisioning of a network infrastructure makes most sense when the network is designed and built using clearly defined, repeatable patterns. It makes less sense when a network infrastructure is using a non-uniform design. Although you can use Ansible AVD to provision a non-uniform network while harvesting some benefits, you’re creating a lot of manual work and customization of Ansible AVD to achieve what you need, essentially making it much harder to maintain going forward.
+
+Below is a table highlighting a few things to consider before jumping head first into automation with Ansible AVD. The table is by no means conclusive and the answer to whether or not to use Ansible AVD quickly becomes “it depends”, but hopefully the table can provide a baseline for what to consider.
+
+
+Consider using Ansible AVD when|Reconsider using Ansible AVD when
```

Comment:
I believe we should talk about culture, which is one of the major stoppers I am seeing in some customers. We as network engineers love to go onto CLI and configure features. Going for AVD is a big change also in mentality. I know it is very subjective, but if this is not considered I am afraid we can end seeing customers going for a part of AVD only and doing other things manually and this can be a mess at the end while also ruining customer experience.

Reply:
Reworked in latest commit